inputfiles/overridingTypes.json

```diff
@@ -2290,6 +2290,57 @@
                         }
                     }
                 }
+            },
+            "Animation": {
+                "name": "Animation",
+                "events": {
+                    "event": [
+                        {
+                            "name": "finish",
+                            "type": "AnimationPlaybackEvent"
+                        },
+                        {
+                            "name": "cancel",
+                            "type": "AnimationPlaybackEvent"
+                        }
+                    ]
+                }
+            },
+            "KeyframeEffect": {
+                "name": "KeyframeEffect",
+                "constructor": {
+                    "override-signatures": [
+                        "new(target: Element | null, keyframes: object | null, options?: number | KeyframeEffectOptions): KeyframeEffect",
```

Comment:
This is to manually remove a reference to CSSPseudoElement that nobody implements. (Currently only Firefox Nightly has it.)